core/src/main/java/hudson/model/ManageJenkinsAction.java

```diff
@@ -52,4 +62,29 @@ public String getDisplayName() {
     public String getUrlName() {
         return "/manage";
     }
+
+    @Override
+    public Object getStaplerFallback() {
+        return Jenkins.get();
```

Comment:
This has been causing some confusion, because now there are two URLs to access every admin-type page, with and without the `manage/` infix, and various pages link to one or the other inconsistently.

Reply:
No good, compatible way around this, unfortunately.

Reply:
Can we not serve 302s to the `/manage/*` versions?